### PR TITLE
Update index.d.ts

### PR DESCRIPTION
**Pull request by IvanKalinin**
Declare types for `off` method.

### CODE REVIEW

File: src/index.d.ts [hunk 1]
```diff
@@ -11,7 +11,8 @@ interface Props {
 }
 
 declare class Flickity extends React.Component<Props, any> {
-    public on(eventName: string, listener: Function): void;
+        public on(eventName: string, listener: Function): void;
```
Comment (yaodingyd) [inline src/index.d.ts]: don't make unrelated change

**Comment by IvanKalinin** [comment]
@yaodingyd
A friendly reminder in case you missed it. I resolved your comment and believe it's ready to be merged.
